### PR TITLE
(minor) NewsDownloader: Koreader to KOReader

### PR DESCRIPTION
**Pull request by Frenzie**

### CODE REVIEW

File: plugins/newsdownloader.koplugin/main.lua [hunk 1]
```diff
@@ -248,7 +248,7 @@ end
 
 function NewsDownloader:setCustomDownloadDirectory()
     UIManager:show(InfoMessage:new{
-       text = _("To select a folder press down and hold it for 1 second. \n\n Please restart Koreader afterwards for the applied changes to take effect.")
+       text = _("To select a folder press down and hold it for 1 second.\n\nPlease restart KOReader afterwards for the changes to take effect.")
```
Comment (Hzj-jie) [inline plugins/newsdownloader.koplugin/main.lua]: This is a little bit strange: why do we need to restart KOReader for the changes to take effect?

Comment (Frenzie) [inline plugins/newsdownloader.koplugin/main.lua]: @mwoz123 ?

Comment (mwoz123) [inline plugins/newsdownloader.koplugin/main.lua]: I'd some problems doing that without restart. 
So for now it requires restart.

**Comment by mwoz123** [comment]
Problem addressed: #2901 ;)

**Pull request merged**